src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/PropertyTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator\Traits;
+
+trait PropertyTrait
+{
+    /**
+     * Sets a specific property.
+     *
+     * @param string $name
+     * @param mixed  $value
+     *
+     * @return $this
+     */
+    final public function property($name, $value)
+    {
+        $this->definition->setProperty($name, static::processValue($value, true));
```

Comment:
`abstract protected static function processValue` to each trait calling it?

Reply:
bof, that's a trait, not a type, there is no contract here.